guides/common/modules/ref_predefined-roles-available-in-project.adoc

```diff
@@ -1,6 +1,5 @@
-[id="Predefined_Roles_{context}"]
+[id="Predefined_Roles__Available_in_Project_{context}"]
```

Comment:
```suggestion
[id="Predefined_Roles_Available_in_{project-context}_{context}"]
```
We actually have a variable you can use in IDs and links! :)

Reply:
Thanks! I was looking for {projectid}. I saw other variables designed for IDs to use that syntax. Will do my best to remember this one.